Sources/Mnemonic.swift

```diff
@@ -5,56 +5,58 @@
 // file LICENSE at the root of the source code distribution tree.
 
 import Foundation
+import TrezorCrypto
 
 public final class Mnemonic {
-    /// Encodes a message using a list of words.
-    public static func encode(message: String) -> [String] {
-        var out = [String]()
-        let n = UInt(mnemonicWords.count)
+    /// Generates a menmoic string with the given strength in bits.
+    ///
+    /// - Precondition: `strength` is a multiple of 32 between 128 and 256
+    /// - Parameter strength: strength in bits
+    /// - Returns: mnemonic string
+    public static func generate(strength: Int) -> String {
+        precondition(strength % 32 == 0 && strength >= 128 && strength <= 256)
```

Comment:
Would it be a good practice to have `enum`? I would assume most people would use 64, 128, 256.